cmd/idmd/db/db.go

```diff
@@ -0,0 +1,85 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright 2021 The LibreGraph Authors.
+ */
+
+package db
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/libregraph/idm/cmd/idmd/db/export"
+	"github.com/libregraph/idm/cmd/idmd/db/load"
+	"github.com/spf13/cobra"
+)
+
+var (
+	BoltDBFile = ""
+	LDAPBaseDN = ""
+	LogLevel   = "info"
+	InputFile  = ""
+)
+
+func CommandDB() *cobra.Command {
+	dbCmd := &cobra.Command{
+		Use:   "db [...args]",
+		Short: "Utility commands related to the BoltDB Database Handler",
```

Comment:
DB vs BoltDB - i think this should be split up so that the "db" command is not a "boltdb" command. Maybe nest it into `db boltdb ..`.